framework/branch-key-store.md

```diff
@@ -109,6 +166,42 @@ If the [AWS KMS Configuration](#aws-kms-configuration) designates MRK ARN compat
 then two ARNs are compatible if they are equal in all parts other than the region.
 That is, they are compatible if [AWS KMS MRK Match for Decrypt](aws-kms/aws-kms-mrk-match-for-decrypt.md#implementation) returns true.
 
+If the [AWS KMS Configuration](#aws-kms-configuration) is Discovery or MRDiscovery,
+no comparison is ever made between ARNs.
```

Comment:
I think you mean something like "The ARN stored in the key store is used..." or something. You are trying to get at the fact that there is nothing to compare?

I mean in the case of MRDiscovery, there is a kind of comparison right, because if the key is an MRK, then we would modify it to the correct region right?

Reply:
I believe MRDiscovery's logic is covered elsewhere.

What wording would you suggest for this section?